### ISSUE TITLE
java.lang.IllegalStateExceptionTransmitter.kt:156 cannot make a new request because the previous response is still open: please call response.close()

### ISSUE THREAD

**Issue by ShaharyarMaroof** [plaid/react-native-plaid-link-sdk#78]
```
java.lang.IllegalStateException cannot make a new request because the previous response is still open: please call response.close() 
    Transmitter.kt:156 okhttp3.internal.connection.Transmitter.newExchange$okhttp
    ConnectInterceptor.kt:35 okhttp3.internal.connection.ConnectInterceptor.intercept
    RealInterceptorChain.kt:112 okhttp3.internal.http.RealInterceptorChain.proceed
    RealInterceptorChain.kt:87 okhttp3.internal.http.RealInterceptorChain.proceed
    CacheInterceptor.kt:82 okhttp3.internal.cache.CacheInterceptor.intercept
    RealInterceptorChain.kt:112 okhttp3.internal.http.RealInterceptorChain.proceed
    RealInterceptorChain.kt:87 okhttp3.internal.http.RealInterceptorChain.proceed
    BridgeInterceptor.kt:84 okhttp3.internal.http.BridgeInterceptor.intercept
    RealInterceptorChain.kt:112 okhttp3.internal.http.RealInterceptorChain.proceed
    RetryAndFollowUpInterceptor.kt:71 okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept
    RealInterceptorChain.kt:112 okhttp3.internal.http.RealInterceptorChain.proceed
    RealInterceptorChain.kt:87 okhttp3.internal.http.RealInterceptorChain.proceed
    RNFetchBlobReq.java:385 com.RNFetchBlob.RNFetchBlobReq$2.intercept
    RealInterceptorChain.kt:112 okhttp3.internal.http.RealInterceptorChain.proceed
    RealInterceptorChain.kt:87 okhttp3.internal.http.RealInterceptorChain.proceed
    RealCall.kt:184 okhttp3.RealCall.getResponseWithInterceptorChain
    RealCall.kt:136 okhttp3.RealCall$AsyncCall.run
    ThreadPoolExecutor.java:1167 java.util.concurrent.ThreadPoolExecutor.runWorker
    ThreadPoolExecutor.java:641 java.util.concurrent.ThreadPoolExecutor$Worker.run
    Thread.java:764 java.lang.Thread.run
```

@zsweigart  I'm getting this issue on Android. If I relaunch the app after crash it does not happen again.

**Comment by zsweigart** [comment]
Thanks for the report, I'll take a look

**Comment by ShaharyarMaroof** [comment]
Ah, so I have a flow where I can connect a bank account from plaid.
1. I connect a bank account using Plaid SDK.
2. Relaunch the app and come back to the Plaid Screen. // The app crashed here.
3. Relaunch after crash, and come back to the Plaid Screen, the app does not crash.

**Comment by ShaharyarMaroof** [comment]
@zsweigart any idea what the issue is? and is there any workaround?

**Comment by zsweigart** [comment]
Hi @ShaharyarMaroof we have been unable to reproduce.  A few more questions:

1. What device/emulator are you running on?
2. What OS version is the device/emulator using?
3. What environment of Plaid are you connecting to?
4. From step 1->2 do you mean you  are going through the link flow successfully, then killing the app then reopening it or are you just backgrounding it and reopening it?

**Comment by ShaharyarMaroof** [comment]
Hey @zsweigart I tried it just now and its not reproducing at my end as well. Earlier it was happening consistently, I think it must've been some config issue. But I was using the following when it was consistently reproducing
Devices:
1. Pixel 3 emulator
2. Samsung Galaxy A70 and Samsung Galaxy A71

OS: Android 9

Plaid Env: sandbox

Yes, I successfully connected the bank account via Plaid, then killed the app and relaunched it.
